docs/modules/ROOT/pages/documentation/payara-server/h2/h2.adoc

```diff
@@ -1,9 +1,8 @@
 [[h2-database]]
 = H2 Database
 
-H2 is a Java based Database which will replace Derby as the default database 
-in Payara 5. Derby will be still included with Payara 5 but will no 
-longer be the default database.
+H2 is a Java-based Database which replaced Derby as the default database in Payara 5. Derby itself as a database that can be started by Payara Server Enterprise, is removed in version 5.201.
```

Comment:
Better to say: "Derby as a database service that can be started and managed by Payara Server Enterprise, has been removed starting from version 5.201"